Networking/Networking/Model/ShippingLabel/Packages/Predefined package/WooShippingSavedPredefinedPackage.swift.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+import Codegen
+import WooFoundation
+
+/// Container struct for a predefined Shipping Label Packages for the WooCommerce Shipping extension with group title and provider identifier.
+///
+public struct WooShippingSavedPredefinedPackage: Equatable, GeneratedFakeable, Identifiable {
```

Comment:
I created this "container" struct to have `WooShippingPredefinedPackage` but also group title and provider ID so we can use that in the saved list UI and for selection

Reply:
`WooShippingPredefinedOption` has `predefinedPackages: [WooShippingPredefinedPackage]` and when we favorite/save a package we favorite/save one of those `WooShippingPredefinedPackage`.

So in theory I could have `WooShippingPredefinedOption` and only have 1 `WooShippingPredefinedPackage` in that array or have this "container" structure just to hold `WooShippingPredefinedPackage` with title and provider ID.

Reply:
Got it, thanks!